client/src/components/SignUp.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState, useEffect } from 'react';
+import { BrowserRouter, Route, Link, Redirect, Switch } from "react-router-dom";
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
```

Comment:
We can simplify imports to one line here, see: https://material-ui.com/guides/minimizing-bundle-size/